web/src/main/java/org/cbioportal/web/util/appliers/PatientTreatmentTargetFilterApplier.java

```diff
@@ -0,0 +1,20 @@
+package org.cbioportal.web.util.appliers;
+
+import org.cbioportal.model.ClinicalEventKeyCode;
+import org.cbioportal.web.parameter.StudyViewFilter;
+import org.cbioportal.web.parameter.filter.AndedPatientTreatmentFilters;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PatientTreatmentTargetFilterApplier extends AbstractPatientTreatmentFilter {
+
+    @Override
+    protected AndedPatientTreatmentFilters getFilters(StudyViewFilter filter) {
+        return filter.getPatientTreatmentTargetFilters();
+    }
+
+    @Override
+    protected ClinicalEventKeyCode getCode() {
+        return ClinicalEventKeyCode.AgentTarget;
```

Comment:
So taking patient data as an example
`Agent` -> `PatientTreatment`
`AgentClass` -> `PatientTreatmentGroup`
`AgentTarget` -> `PatientTreatmentTarget`
Is this correct?

Reply:
Yup